pkg/server/profiles.go

```diff
@@ -146,15 +140,40 @@ func (s *ProfilesServer) GetProfileValues(ctx context.Context, msg *pb.GetProfil
 		return nil, fmt.Errorf("failed to get HelmRepository %q/%q", s.HelmRepoNamespace, s.HelmRepoName)
 	}
 
-	sourceRef := helmv2beta1.CrossNamespaceObjectReference{
```

Comment:
This was actually never used. And has no function in any case anyways because only the profile name and version matter.